db/academyPortal.sql

```diff
@@ -25,7 +25,7 @@
 
 DROP TABLE IF EXISTS `admins`;
 
-CREATE TABLE `admins` (
+CREATE TABLE users (
   `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
   `email` varchar(255) NOT NULL DEFAULT '',
   `password` varchar(65) NOT NULL DEFAULT '',
```

Comment:
this needs to increase as it may truncate the hashed password (use 256 length)

Reply:
Done.